features/lorawan/LoRaRadio.h

```diff
@@ -18,7 +18,7 @@
 #ifndef LORARADIO_H_
 #define LORARADIO_H_
 
-#include "platform/Callback.h"
+#include "Callback.h"
```

Comment:
it is common in our codebase to use prefixes in the includes. This won't fix the issue completely (using prefixes in drivers/platform etc). What compiler error are you getting?

@hasnainvirk can you reproduce this? why is this just for this header file?